modules/k8s/pod.go

```diff
@@ -137,5 +137,10 @@ func WaitUntilPodAvailableE(t testing.TestingT, options *KubectlOptions, podName
 
 // IsPodAvailable returns true if the pod is running.
```

Comment:
Can you update the docs to reflect the update? Also update the docs for `WaitUntilPodAvailableE`.

Reply:
Of course , i already added this changes in commit 120673a.